src/main/java/org/jabref/gui/customentrytypes/EntryCustomizationDialog.java

```diff
@@ -342,7 +336,7 @@ private void deleteType(String name) {
                 }
             }
             EntryTypes.removeType(name, bibDatabaseMode);
-            updateEntriesForChangedTypes(Arrays.asList(name.toLowerCase(Locale.ENGLISH)));
+            updateEntriesForChangedTypes(Collections.singletonList(name.toLowerCase(Locale.ENGLISH)));
```

Comment:
You are aware that Collections,singletonList is immutable?

Reply:
Thanks for the hint, I was actually not aware of it. In this case, however, it does not matter since `updateEntriesForChangedTypes` only invokes `contains` on the list.